torchbox/torchbox.com
# Message to display when there are no events

[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1400769730)

### Description of Changes Made

This adds a RichTextField to the EventIndexPage, so that content editors can specify a message to use when there are no events.

### How to Test

Edit the text in the `no_events_message` field on an `EventIndexPage`, and see the changes reflected in the template.

### Screenshots

<details>
  <summary>Expand to see more</summary>

![Screenshot from 2024-04-12 10-34-33](https://github.com/torchbox/torchbox.com/assets/7713776/7cd60e19-264e-453f-a9f0-83347b50775e)

</details>

> [!NOTE]  
> There might be some styling required, as you'll see from the screenshot above that the link is not styled

### MR Checklist


#### Unit tests


#### Documentation


#### Browser testing

  - Latest version of Chrome on Ubuntu
  - Latest version of Firefox on Ubuntu

#### Data protection


#### Accessibility


#### Sustainability


#### Pattern library
